website/src/pages/documents/document.page.tsx

```diff
@@ -163,12 +175,9 @@ const TranslationTab = ({ doc }: { doc: Document }) => {
     setCurrContents: selectAndShowWord,
   }
 
-  const [phoneticRepresentation, _setPhoneticRepresentation] =
-    useState<PhoneticRepresentation>(selectedPhonetics())
+  const phoneticRepresentation = useContext(preferencesContext).phonRep
```

Comment:
Make a hook inside whatever file you make the `PreferencesProvider` component, a new hook called `usePreferences` that wraps this useContext call. Generally, convention is that you never export a context directly, you only use it from abstractions like a custom provider component or custom hook. Also, you only need to call the hook once here, like `const { viewMode, phoneticRepresentation } = usePreferences()`.